cumulusci/core/config/sfdx_org_config.py

```diff
@@ -25,7 +25,7 @@ def sfdx_info(self):
         username = self.config.get("username")
         assert username is not None, "SfdxOrgConfig must have a username"
 
-        self.logger.info(f"Getting org info from Salesforce CLI for {username}")
+        # self.logger.info(f"Getting org info from Salesforce CLI for {username}")
```

Comment:
@jain-naman-sf Can we suppress this line only in case of --json option is used?

Reply:
@jkasturi-sf No now it will be suppressed without —json option as well. This fix will be working in both cases